migrations/2023/Version20230912132551.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230912132551 extends AbstractMigration
+{
+  public function getDescription(): string
+  {
+    return '';
+  }
+
+  public function up(Schema $schema): void
+  {
+    // this up() migration is auto-generated, please modify it to your needs
+    $this->addSql('ALTER TABLE program ADD popularity DOUBLE PRECISION DEFAULT \'0\' NOT NULL');
+    $this->addSql('ALTER TABLE studio_activity CHANGE type type ENUM(\'comment\', \'project\', \'user\')');
+    $this->addSql('ALTER TABLE studio_user CHANGE role role ENUM(\'admin\', \'member\'), CHANGE status status ENUM(\'active\', \'banned\', \'pending_request\')');
+    $this->addSql('ALTER TABLE user_like_similarity_relation CHANGE similarity similarity NUMERIC(4, 3) DEFAULT \'0\' NOT NULL');
+    $this->addSql('ALTER TABLE user_remix_similarity_relation CHANGE similarity similarity NUMERIC(4, 3) DEFAULT \'0\' NOT NULL');
```

Comment:
Those 4 lines can be removed, they have no effect, and will be in every migration. There must be a misconfiguration in our settings. You can remove them